tests/analyzer_tech_debt.txt

```diff
@@ -11,12 +10,9 @@
 01244_optimize_distributed_group_by_sharding_key
 01268_mv_scalars
 01268_shard_avgweighted
-01287_max_execution_speed
```

Comment:
No matter how many times this test is removed from the list, it's flaky with the analyzer and the analyzer only and it needs fixing. See https://s3.amazonaws.com/clickhouse-test-reports/56984/ff438112e95568bb322c9ff8283e6b7f2e1b412a/stateless_tests__release__analyzer_.html

5 failures in 24 hours. All with the analyzer